packages/widget/src/pages/SwapExecutionPage/SwapExecutionPageRouteDetailed.tsx

```diff
@@ -191,6 +193,11 @@ export const SwapExecutionPageRouteDetailed = ({
           index={0}
         />
         {renderOperations}
+        {isGasStationTx && 
+        <StyledGasStationTxText>
+          Transactions from EVM to Babylon have gas provided automatically.
+        </StyledGasStationTxText>
+        }
```

Comment:
As far as I know we provide gas if they have 0 BABY on Babylon. might want to change the wording "if the user don't have BABY" or we can check BABY balance first and if they don't have BABY we can show this warning

Reply:
pushed a fix to check balance. Its not ideal however if we've only fetched balances for EVM because source must be EVM. But better to display the message in more cases than not